### DISCUSSION TITLE
Discussing the posibility of adding DUAL GRID tilemaps.

### DISCUSSION THREAD

**Discussion by CarpenterBlue** [godotengine/godot-proposals#11226] [2D]
Today I stumbled on youtube upon this little gem:
[Draw fewer tiles - by using a Dual-Grid system! - by jess codes](https://www.youtube.com/watch?v=jEWFSv3ivTg)
And while implementing it seems rather easy, I think it would be extremely beneficial if Godot had this as an another kind of tilemap.
I am talking about it here before I turn it into proper proposal.

**Comment by tetrapod00** [comment]
Thanks for the idea! There is actually already a discussion (https://github.com/godotengine/godot-proposals/discussions/10422) and a full proposal (https://github.com/godotengine/godot-proposals/issues/10567) for this. In the proposal, a TileMap maintainer commented and made a separate proposal for an implementation (https://github.com/godotengine/godot-proposals/issues/10572). 

I'm closing this as a duplicate, I would suggest sharing any thoughts you have in one of the existing discussions or proposals (and feel free to share your support with a 👍 reaction on the proposals).
